### PR TITLE
Gif block - add example

### PR DESCRIPTION
**Pull request by MichaelArestad**
* props @gravityrail

Master issue: #13510

#### Changes proposed in this Pull Request:

* Added an example preview to the block in the Inspector.

#### Before
image

#### After
image

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* Adding support for a new feature

#### Testing instructions:

* Create a new post
* Click the + to open the Inserter
* Hover over Gif

#### Proposed changelog entry for your changes:

* None needed

### CODE REVIEW

**Comment by joanrho** [comment]
When I tested earlier today, I got this animated "OOPS" GIF which I think might be an error in loading the WP logo gif properly?

Screen Shot 2019-10-01 at 12 30 44 PM

**Comment by MichaelArestad** [comment]
@joanrho fixed. It was a typo. My typo.

**Comment by joanrho** [comment]
@MichaelArestad Thanks for the fix. I see the rotating logo now. Looks great and works as expected!

**Review by jeherve — APPROVED** [review]
This looks nice, it's simple enough. I like it! Merging.

**Comment by keoshi** [comment]
NOICE!

File: extensions/blocks/gif/index.js [hunk 1]
```diff
@@ -58,4 +58,12 @@ export const settings = {
 	},
 	edit,
 	save: () => null,
+	example: {
+		attributes: {
+			align: 'center',
+			giphyUrl: 'https://giphy.com/embed/fxKWgoOG9hzPPkE1oc',
+			paddingTop: '100%',
+			searchText: __( 'WordPress', 'jetpack' ),
```
Comment (simison) [inline extensions/blocks/gif/index.js]: Does this need to be translated?

Comment (MichaelArestad) [reply extensions/blocks/gif/index.js]: I suppose not since it doesn't visibly show up. Will remove.

**Pull request merged**